### DISCUSSION TITLE
how to install master branch for coc-prettier

### DISCUSSION THREAD

**Discussion by laoshaw** [neoclide/coc.nvim#4712] [General]
coc-prettier was just fixed for prettier 3.0 new version but how can I use it? CocInstall https://github.com/neoclide../coc-prettier complains and I'm stuck at v9.3.1. I know that sounds like a coc-prettier but the general question is how to cocinstall head-of-some-master-branch for its extensions.

**Comment by weirongxu** [comment]
I just released a temporary V3 version, you can try it `:CocInstall @raidou/coc-prettier-v3`

**Comment by laoshaw** [comment]
Thanks, the error message seems gone.

**Comment by chemzqm** [comment]
A release was published for coc-prettier, https://www.npmjs.com/package/coc-prettier
